pilot/notifications.rst

```diff
@@ -0,0 +1,40 @@
+.. _notifications:
+
+Notifications
+=======================================
+
+As a creator / owner of resources inside Ronin you may receive Email based notification depending on their state.
+These intend to alert you of possible under utilization of resources to help avoid unnecessary project spend.
+
+.. note::
+    All notifications will come from the address **noreply@ronin.cloud**, always be cautious when following links from external addresses!
+
+.. _instance_utilization:
+
+Instance Utilization
+---------------------------------------
+
+If you have received an email titled "Ronin Under Utilization Alert!" this means that Ronin has noticed an :term:`instance` in your project has gone for more than more than 24 hours with under 10% CPU utilization.
+This is designed to ensure you've not accidentally left a machine running idle. Unlike on-premise VM's that run 24/7 we recommend you shut down your instances when not in use, much like you would your own PC. 
```

Comment:
```suggestion
This is designed to ensure you've not accidentally left a machine running idle. Unlike on-premise VMs that run 24/7 we recommend you shut down your instances when not in use, much like you would your own PC. 
```

Reply:
Worth noting that lets project budget stretch further if only use machines when needed.

Reply:
```suggestion
This is designed to ensure you've not accidentally left a machine running idle. Unlike on-premise VMs that run 24/7 we recommend you shut down your instances when not in use, much like you would your own PC. 

Only instances that are running are subject to the hourly usage rates however you will still be charged for the provision of storage as it explained below.
```